deploy/rbac.yaml

```diff
@@ -105,3 +105,8 @@ roleRef:
   kind: Role
   name: percona-server-mongodb-operator
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: percona-server-mongodb-operator-workload
```

Comment:
Why do we need to create this account ?

Reply:
it seems an overkill for me also)
I will remove it